zips/draft-ecc-zbloc.md

```diff
@@ -0,0 +1,268 @@
+
+# Specification
+
+## zBloc Membership
+
+The zBloc would distribute decision-making authority for both Zcash governance and funding decisions, potentially to any number of Constituencies.
+
+A zBloc Constituency is a group representing a trusted subsection of the Zcash community.
+
+The [Initial Structure] section below proposes a set of initial Constituencies, but the model allows for changes over time, enabling the zBloc to evolve. New zBloc Constituencies may be added as they gain trust and removed if they are no longer operating or trust is eroded. The means for this is described in the [Governance Decisions] section below.
+
+zBloc Constituencies may be granted different levels of authority, receiving less or more say over time. This is done by allocating or removing "Voting Units" to reflect the amount of voting power allocated to each Constituency. For example, if the community wishes to increase the weight of coin-holder voice over time, it could increase the number of Voting Units allocated to coin-holder representatives, which would dilute the relative power of other Constituencies.
+
+### Decision-making policies
+
+To be considered a Constituency, each must pre-commit to an internal policy for decision-making that all other Constituencies approve of (i.e. greater than one decision maker, means of assessing community sentiment, etc.). This could be by polling, majority vote within the Constituency, coin-holder signaling, technical feasibility, mission alignment, etc.
+
+In order for a Constituency to change its decision-making policy, the change MUST be expressed as a [Governance Proposal] and Approved before it comes into effect. The Constituency that is proposing to change its policy is eligible to vote, but it MUST make its decision according to its *pre-existing* policy (if it does not abstain).
+
+## Initial Structure
+
+The initial group of Constituencies will be (if they are willing and approved by the others and by the Zcash community):
+
+* Coin-holders
+* Electric Coin Company
+* Zcash Foundation
+* Shielded Labs
+* Zcash Community Grants
+* Zcash Core Engineers
+* ZecHub
+
+Each Constituency initially receives three Voting Units. This MAY be changed by a Governance Proposal.
+
+Constituencies other than the Coin-holder Constituency above are called Organizational Constituencies. The Organizational Constituencies will together elect a representative responsible for establishing a clear process and mechanism for gathering coin-holder sentiment, which will then be used as the decision-making process of the Coin-holder Constituency.
```

Comment:
One possibility that this doesn't take into account is that an organic coin-holder constituency may choose to self-organize; the way that this is worded makes it sounds like the Organization Constituencies will choose the decision-making process instead of just the mechanism for determining coin-holder sentiment.
```suggestion
Constituencies other than the Coin-holder Constituency above are called Organizational Constituencies. The Organizational Constituencies will together elect a representative responsible for establishing a clear process and mechanism for gathering coin-holder sentiment, which will then be used to determine the will of the Coin-holder Constituency.
```